src/monitorCloudFormationStack.ts

```diff
@@ -0,0 +1,82 @@
+import config from "./config";
+import { DynamoDB } from "aws-sdk";
+import { StackJanitorStatus } from "stackjanitor";
+import { logger } from "./helpers";
+import { DocumentClient } from "aws-sdk/lib/dynamodb/document_client";
+
+const documentClient = new DynamoDB.DocumentClient();
+
+export enum RequestType {
+  CREATE = "CreateStack",
+  UPDATE = "UpdateStack"
+}
+
+export enum Const {
```

Comment:
/nit rename `Const` to something meaningful like `StatusMessage`

Reply:
done